src/openutm_verification/simulator/geo_json_flight_simulator.py

```diff
@@ -267,11 +268,14 @@ def generate_aircraft_states(
         config=GeoJSONFlightsSimulatorConfiguration(
             reference_time=arrow.utcnow(),
             geojson=geojson_data,
-            altitude_of_ground_level_wgs_84=200,
+            altitude_of_ground_level_wgs_84=500,
```

Comment:
Is there a reason why this was changed? My recommendation is to keep keep it below 500